### DISCUSSION TITLE
Whitelist Capacitor Android / ios

### DISCUSSION THREAD

**Discussion by TinyLightBold** [dexie/Dexie.js#1947] [General]
I am working atm on an application build in Quasar Vite with dexie for the database and dexie-cloud as Cloud solution.
I had issues making myself clear what the Origin was for Android apps and possible Ios apps to make the Cloud work.

Could not find it anywhere how to do it. 
This morning I found out it was actually very simple:
for Android: `npx dexie-cloud whitelist https://localhost` (this I am sure!)
for Ios Capacitor it could be: `npx dexie-cloud whitelist capacitor://localhost` (don't have a Mac to try this out)

Let's be clear, this is for the Capacitor version of application build. For normal development it could be different. (not my area)

For people who are in search what Origin is needed to Whitelist to make Cloud work on Android/Ios, this could make their lives easier ;).

Could be added to https://dexie.org/cloud/docs/cli#whitelist.

**Comment by dfahlander** [comment]
Thank you very much! I've updated the docs. I also added a small section about how to detect origin on the client by using `location.origin`. https://dexie.org/cloud/docs/cli#detecting-which-address-to-whitelist
